health-services/project-factory/src/server/service/dataManageService.ts

```
@@ -120,7 +120,10 @@ const getBoundaryDataService = async (
 
 const createDataService = async (request: any) => {
 
-    const localizationMap = await getLocalizedMessagesHandler(request, request?.body?.ResourceDetails?.tenantId);
+    const { hierarchyType } = request?.body?.ResourceDetails;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)